pkg/config/nerdctl_config_applier.go

```diff
@@ -139,7 +141,7 @@ func (nca *nerdctlConfigApplier) Apply(remoteAddr string) error {
 		return fmt.Errorf("failed to update the nerdctl config file: %w", err)
 	}
 
-	if err := updateEnvironment(sftpFs, user); err != nil {
+	if err := updateEnvironment(sftpFs, nca.hostUser); err != nil {
```

Comment:
A comment explaining why we need to update the host users `.bashrc` file would be useful.

Reply:
I think the comment for `updateEnvironment` has already explained it well?

Reply:
Yeah that part explains why we set `DOCKER_CONFIG` in `.bashrc`. I was referring to the fact that we need to update the host users `/home/<user>.linux/.bashrc` file vs root's  because `sudo -E` does not source `/root/.bashrc`.

Reply:
updated